### DISCUSSION TITLE
Can i get last scheduled message? Can i schedule message using bot?

### DISCUSSION THREAD

**Discussion by v4ndn** [python-telegram-bot/python-telegram-bot#2889] [Q&A]
Hi there! I want to get last scheduled message, and get a posting time from it, then i will plus 3h to posting time and schedule a new message. How i can do it?

**Comment by Bibo-Joshi** [comment]
Hi. I don't understand what your question is here. Please have a look at [this wiki page](../wiki/Ask-Right) as well as [this one](../wiki/Extensions-–-JobQueue).
